src/components/CommentList/CommentList.jsx

```diff
@@ -1,3 +1,11 @@
-export const CommentList = () => (
-  <>Put the list here</>
+import './CommentList.scss';
+
+import { CommentInfo } from '../CommentInfo';
+
+export const CommentList = ({ comments }) => (
+  <div className="CommentList">
+    {comments.map(comment => (
+      <CommentInfo comment={comment} />
```

Comment:
it's a bad practice to write key not at root tag, rewrite it to: `<CommentInfo comment={comment} key={comment.id} />`